modules/eggnog/download/main.nf

```diff
@@ -0,0 +1,57 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process EGGNOG_DOWNLOAD {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::eggnog-mapper=2.1.6" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/eggnog-mapper:2.1.6--pyhdfd78af_0"
+    } else {
+        container "quay.io/biocontainers/eggnog-mapper:2.1.6--pyhdfd78af_0"
+    }
+
+    input:
+    val(meta)
```

Comment:
do we need this? I've seen other modules downloading reference genomes that had no input at all.

Reply:
Nope not needed and happy to remove